airflow/contrib/kubernetes/kubernetes_request_factory/pod_request_factory.py

```diff
@@ -60,3 +60,60 @@ def create(self, pod):
         self.extract_annotations(pod, req)
         self.extract_affinity(pod, req)
         return req
+
+
+class ExtractXcomPodRequestFactory(KubernetesRequestFactory):
+
+    XCOM_MOUNT_PATH = '/airflow/xcom'
+    SIDECAR_CONTAINER_NAME = 'airflow-xcom-sidecar'
+    """
+    Request generator for a simple pod.
+    """
+    _yaml = """apiVersion: v1
+kind: Pod
+metadata:
+  name: name
+spec:
+  volumes:
+    - name: xcom
+      emptyDir: {{}}
+  containers:
+    - name: base
+      image: airflow-worker:latest
+      command: ["/usr/local/airflow/entrypoint.sh", "/bin/bash sleep 25"]
+      volumeMounts:
+        - name: xcom
+          mountPath: {xcomMountPath}
+    - name: {sidecarContainerName}
+      image: python:3.5-alpine
```

Comment:
Thank you for keeping this image light-weight. There was some ongoing work to switch the airflow pod to Alpine that we should really pick up again.